bugbug/repository.py

```diff
@@ -216,6 +222,25 @@ def download_commits(repo_dir, date_from):
     r.raise_for_status()
     COMPONENTS = r.json()
 
+    components_touched = {}
+
+    for commit in commits:
+        comp = set('::'.join(COMPONENTS[fl]) for fl in commit.files.decode('utf-8').split('|') if fl in COMPONENTS)
+        for cp in comp:
+            if cp in components_touched:
+                components_touched[cp] += 1
+            else:
+                components_touched[cp] = 1
+
+        COMP_TOUCHED_PREV[commit] = sum(components_touched[cp]-1 for cp in comp)
```

Comment:
Hi. Just noticed failing Travis check. :)
You need to add whitespace around arithmetic operator on this line and on line 239 (in 2 places) to have it passed.

Reply:
Hi! Thanks for pointing it out.

I have this thing in my notice. Actually I was waiting for @marco-c  to review this PR. So that if there'd be any other changes, I'll fix this missing whitespace along with those so as to avoid some extra commits for minor syntax changes.

Reply:
When you notice some issue, feel free to fix it right away. It's OK if there are multiple commits, indeed it's better to have one commit for each thing you do.

Reply:
Okay.